pages/emoticons_page.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: UTF-8 -*-
+
+from .base_page import Page
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+import os
+
+
+class EmoticonsPage(Page):
+
+    url = '/emoticons'
+
+    def go_to_filepicker(self):
+        self.context.driver.find_element_by_id('filepicker').click()
+
+    def set_emoticon_name(self):
+        self.context.wait.until(EC.visibility_of_element_located((By.ID, 'shortcut_0')))
+        self.context.driver.find_element_by_id('shortcut_0').send_keys('test')
+
+    def upload_image(self):
+        img_path = os.getcwd() + '/doc/behave_web.png'
+        self.context.driver.find_element_by_id('Filedata_0').send_keys(img_path)
+
+    def click_add_emoticon(self):
+        self.context.driver.find_element_by_id('submit_emoticon').click()
+
+    def wait_if_created(self):
+        self.context.wait.until(EC.visibility_of_element_located((By.XPATH, '//td[text()="(behaveweb)"]')))
+
```

Comment:
This is not mistake, this code work fine, I tried it few times local and few times on CI, all time pass, great job! But I want again tell you about method names and about what test method should return, we working with framework Behave, and developers who create it have phylosophy about that how use it, here more, this is interesting https://pythonhosted.org/behave/philosophy.html
I don't use it 100%, but I try, I think you should too.